core/EE_Session.core.php

```diff
@@ -371,6 +383,23 @@ public function lifespan()
     }
 
 
+    /**
+     * @param string $save_state
```

Comment:
`setSaveState()` is just a standard setter with a default value that toggles the save state to dirty,
but you can also use it to reset the save state back to clean:

```php
$this->setSaveState(EE_Session::SAVE_STATE_CLEAN);
```

can't say I agree with renaming it based solely on its default value, because then it makes no sense when passing any other value.

but I will improve (ie: add) some phpdocs explaining what's going on

Reply:
IMO `setDirtySession(false)` seems intuitive- it's setting the session to NOT be dirty, ie clean. But obviously that's a matter of opinion. This is fine as-is too.